README.md

```diff
@@ -305,6 +306,88 @@ RESET_PASSWORD_SECRET_KEY: The secret key for reset password functionality.
 VERIFICATION_SECRET_KEY: The secret key for email or user verification.
 ```
 
+## Production Deployment
+
+### Overview
+
+Deployment is streamlined through **Vercel**, with dedicated buttons for the **Frontend** and **Backend** applications. Both require specific configurations during and after deployment to ensure proper functionality.
```

Comment:
Deployment is streamlined through **Vercel** -> Deploy to **Vercel** is supported.